Editor/Halodi/PackageRegistry/Core/CredentialManager.cs

```diff
@@ -14,6 +14,8 @@ public class NPMCredential
     {
         public string url;
         public string token;
+        public string _auth;
```

Comment:
any particular reason why this field is formatted differently from the others in this class?

Reply:
All of the fields use the exact name that is used in the [.upmconfig.toml](https://docs.unity3d.com/Manual/upm-config.html#upmconfigUser) file, which in this case is `_auth` (with the underscore). A nice side effect is that it is possible to use `nameof(NPMCredential._auth)` to reference the key.

Reply:
I wasn't aware this was reading/writing from that file. Sounds good to me :) Thanks for helping me understand.